src/osvr/Common/ImagingComponent.cpp

```diff
@@ -71,9 +67,7 @@ namespace common {
 
             template <typename T>
             void allocateBuffer(T &, size_t bytes, std::true_type const &) {
-                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(
-                                   cv::fastMalloc(bytes)),
-                               &cv::fastFree);
+                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(malloc(bytes)));
```

Comment:
I will reimplement this part using `std::align`.

I'll also update the client example to show that having a dependency for OpenCV in the ClientKit is completely unnecessary even if the developer uses OpenCV.